packages/studio-plugin/src/parsers/StudioSourceFileParser.ts

```diff
@@ -101,21 +101,14 @@ export default class StudioSourceFileParser {
 
   parseCssImports(): string[] {
     const cssImports: string[] = [];
-    const getAbsolutePath = (source: string) => {
-      if (source.startsWith(".")) {
-        const absPath = upath.join(
-          upath.dirname(this.sourceFile.getFilePath()),
-          source
-        );
-        return absPath;
-      }
-      return source;
-    };
-
     this.sourceFile.getImportDeclarations().forEach((importDeclaration) => {
       const { source } = StaticParsingHelpers.parseImport(importDeclaration);
-      if (source.endsWith(".css")) {
-        cssImports.push(getAbsolutePath(source));
+      const isRelativeCssImport =
+        source.endsWith(".css") && source.startsWith(".");
+      if (isRelativeCssImport) {
+        cssImports.push(
```

Comment:
we weren't previously doing anything with the page CSS imports, right? just curious if we're dropping any support by only adding local CSS imports

Reply:
We were just taking CSS imports directly from the import statement, then storing it into `cssImports`.  The only thing we did before was write these imports back to file.

I think one thing that will be ever so slightly different is if the user has non-relative imports (which we don't support yet) will not be saved to `cssImports` and not written back to the files.  However, these imports should remain, since we just modify the old page file.

This also has an effect on Layouts. Since we don't support non-relative `cssImports` in templates, we don't in Layouts either.